pkg/kubebench/scanner.go

```diff
@@ -26,6 +26,7 @@ import (
 const (
 	kubeBenchVersion       = "0.3.0"
 	kubeBenchContainerName = "kube-bench"
+	masterNodeLabel        = "node-role.kubernetes.io/master"
```

Comment:
Is this const used somewhere? I think we might want to use kube-bench node type detection feature

Reply:
Yes! We could use to identify parameters in kube-bench